pkg/kubectl/cmd/annotate.go

```diff
@@ -189,13 +189,10 @@ func (o AnnotateOptions) RunAnnotate(f cmdutil.Factory, cmd *cobra.Command) erro
 	changeCause := f.Command(cmd, false)
 
 	includeUninitialized := cmdutil.ShouldIncludeUninitialized(cmd, false)
-	b := f.NewBuilder()
-	if o.local {
-		b = b.Local()
-	} else {
-		b = b.Unstructured()
-	}
-	b = b.ContinueOnError().
+	b := f.NewBuilder().
+		Unstructured().
```

Comment:
@juanvallejo  didn't `Unstructured`  attempt to make a remote call and that broke `--local` since it tried to contact the server.

Reply:
Yes, I believe `Unstructured()` makes use of the discovery client to retrieve mappings

Reply:
Yes, and now unstructured fails back to using the innate discovery.